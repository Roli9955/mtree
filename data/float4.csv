id;val
0;0.8948185330924516
1;0.6958975811792261
2;0.8306284712838354
3;0.6037352630875872
4;0.08596736827806628
5;0.030514853992901325
6;0.10191508297240115
7;0.1888094799529988
8;0.440192105433319
9;0.8245091719447365
10;0.3058578336416501
11;0.5184239351945145
12;0.3006333133699367
13;0.00995269379453012
14;0.5537889083886149
15;0.6541721307638212
16;0.703579146103461
17;0.2414374473567168
18;0.21491529775560114
19;0.6075161983138669
20;0.57911859574481
21;0.4098654093761158
22;0.7814681960039016
23;0.8131254603065624
24;0.16676480671544247
25;0.7340385033758363
26;0.16254890014899892
27;0.19295232886919267
28;0.8327486180167614
29;0.5846652995091015
30;0.5867786314244093
31;0.32270767407838374
32;0.3553314932589714
33;0.11333708747610893
34;0.7224770712753972
35;0.7815816010118778
36;0.7629603306314917
37;0.8847863754467472
38;0.1421543147428702
39;0.5223614529351984
40;0.5731994895352139
41;0.6472973145706272
42;0.597281896255015
43;0.1037325208390617
44;0.8273623981594812
45;0.5309553282948671
46;0.5165615333565633
47;0.8649847979623989
48;0.2624672909902538
49;0.7078468479896599
50;0.31194193601136766
51;0.9602651843651729
52;0.6554000250368105
53;0.49841892512097186
54;0.09767341559422804
55;0.7189649341069103
56;0.16687535016381705
57;0.4399034199370979
58;0.2040923736235004
59;0.40753400426759057
60;0.7023557811985417
61;0.518619963262144
62;0.5397582523828452
63;0.23103180804041457
64;0.49760458005387
65;0.13697405183491362
66;0.2484814041147284
67;0.4623333383862458
68;0.5086632680079868
69;0.9388610272821846
70;0.8451925368157797
71;0.6639101450250572
72;0.5609542713189247
73;0.6805039805379365
74;0.1690967692867018
75;0.36133991883218486
76;0.6608223430333952
77;0.5085601159715556
78;0.5735916869369488
79;0.8943461167993261
80;0.5854600265643539
81;0.06924514821614014
82;0.7611175905424998
83;0.3071039870254044
84;0.12752173557858137
85;0.5885572991570144
86;0.2801451857719348
87;0.6432535412145102
88;0.6989431425605266
89;0.7697769091076259
90;0.00811665193379918
91;0.7283393255931425
92;0.5501742092466592
93;0.2690584506043495
94;0.8109395222751618
95;0.23109840926828729
96;0.004027110336066619
97;0.9062843326840707
98;0.9706466816914069
99;0.9703915584519429
100;0.24975382337277663
101;0.8694260811729464
102;0.611249554053054
103;0.8356683467299273
104;0.5426516335347187
105;0.2449160072487132
106;0.17188299842209143
107;0.030406898108399805
108;0.4132296735158454
109;0.527936214101828
110;0.22186600288213065
111;0.20483995943956623
112;0.04660153072421935
113;0.7471115867345353
114;0.8758528155314618
115;0.0015622670848044473
116;0.23761123700684528
117;0.9818911426010271
118;0.438867759567073
119;0.7826381672078122
120;0.04022363322668987
121;0.6871765375232715
122;0.0384217565105901
123;0.27897017980264893
124;0.8637686169983281
125;0.5045719220323719
126;0.4427024305391668
127;0.17424816671506183
128;0.6448050900995904
129;0.07454870885497333
130;0.9771760953253014
131;0.3019416740375317
132;0.012832433122246134
133;0.7911069211168812
134;0.5108947460517805
135;0.2580174484935345
136;0.38224282512598984
137;0.7088700867145294
138;0.9588308568450769
139;0.5636082134640986
140;0.945440025589545
141;0.01507378184297059
142;0.17829584397782527
143;0.20336120896040066
144;0.2614741254637264
145;0.5938151980734405
146;0.12694867739133187
147;0.7734885242124793
148;0.4604380203098065
149;0.03738457559509689
150;0.584122938937319
151;0.18281124262563808
152;0.720096306056895
153;0.26645800927567653
154;0.8381840191714759
155;0.6925393593982659
156;0.7960384103425672
157;0.7392917415913988
158;0.5316275178265263
159;0.3876379946498598
160;0.11884624945774669
161;0.9414569537284772
162;0.5295772421292884
163;0.5332414124638254
164;0.29078844361029743
165;0.9331924747277592
166;0.363817063428329
167;0.4789885184910597
168;0.6128208620211019
169;0.6792343721532226
170;0.05519051181021828
171;0.7625497918246943
172;0.8465431931846124
173;0.8808792824969256
174;0.3042375289967736
175;0.9478194807391879
176;0.23246816944110293
177;0.7457623178968847
178;0.050924024358926734
179;0.199039731126736
180;0.8119741550520958
181;0.3320464885844807
182;0.6052430928917577
183;0.9311343065923835
184;0.932552784540135
185;0.12008281796972264
186;0.05750510605750525
187;0.8459978622525317
188;0.23351118652051284
189;0.8621178167012846
190;0.1055583750418887
191;0.39078011715602223
192;0.8587785158702812
193;0.6087212498517119
194;0.4748954884969645
195;0.21276219508293337
196;0.25211055700676577
197;0.970455845962683
198;0.487947806697285
199;0.0288334584133495
200;0.6060897064238797
201;0.854747846415684
202;0.5168183400246527
203;0.8903647229522086
204;0.559848951557879
205;0.718185950667913
206;0.1190730114779065
207;0.7673935844636465
208;0.9269069762392801
209;0.9112690504663707
210;0.5820491776349765
211;0.8015389046486453
212;0.12541286846306876
213;0.2437180461718469
214;0.8127355989065586
215;0.7620398532901848
216;0.6573030761362643
217;0.8670276781879165
218;0.9284053298299986
219;0.19120092569916192
220;0.27519718294531403
221;0.6263250910985121
222;0.27855686586152617
223;0.47321812379132555
224;0.6903588316126196
225;0.058277899571706815
226;0.3924020539305585
227;0.48814212492727094
228;0.20971488323629228
229;0.22300847659352263
230;0.9799614799176977
231;0.5811742096369986
232;0.3879384005307618
233;0.9359852947074504
234;0.9640881835955919
235;0.666811238277181
236;0.8793516448893193
237;0.05111353916517458
238;0.0987791560246094
239;0.3760791603304664
240;0.9248050096393555
241;0.0012616663407448714
242;0.4739736885516268
243;0.2413161237201784
244;0.9653177204466249
245;0.46579312304637144
246;0.1989763166710794
247;0.04687323461273507
248;0.275542415612664
249;0.42013089024773287
250;0.26429746194584947
251;0.6948631655383924
252;0.01614123819418478
253;0.9282479310044188
254;0.04556473515148207
255;0.9010867501221492
256;0.3948402000325145
257;0.34047010031705816
258;0.8768680912215969
259;0.8873727424940243
260;0.49571460899601827
261;0.45211047541289295
262;0.4834953521989164
263;0.1903224946214951
264;0.9400660314964877
265;0.7605605061832846
266;0.222278627165715
267;0.02439124364498646
268;0.2275335381671778
269;0.14173178930179753
270;0.1778150699482418
271;0.9438277230710459
272;0.7778160964423123
273;0.6394062582850143
274;0.7706423909042487
275;0.6954418967345241
276;0.36841280323855474
277;0.10049904562779355
278;0.9273769676541603
279;0.9097841325448011
280;0.13000976019476906
281;0.2338814819550934
282;0.2941371596862642
283;0.022393795098830993
284;0.7451608340075201
285;0.09077800405988368
286;0.32231641683737455
287;0.19377166001003465
288;0.48198815018956886
289;0.7699665493704164
290;0.8515072053449011
291;0.13567875757892067
292;0.3047882694162415
293;0.6246949048479227
294;0.7508133265514323
295;0.3870044277820256
296;0.9024560754636037
297;0.08023979438151463
298;0.33320060105646887
299;0.026776755958048182
300;0.17891376420782246
301;0.35485033150474443
302;0.6444525969451295
303;0.43201892974827805
304;0.1698531138283048
305;0.973931421061057
306;0.5796568344869446
307;0.7579841689149539
308;0.5248919877845388
309;0.3659568751975053
310;0.7713444974568427
311;0.5331076608541049
312;0.6515764525328911
313;0.4844740121987553
314;0.6107185962832964
315;0.5641822014907774
316;0.388962230288767
317;0.912391852774396
318;0.5182868334310806
319;0.4828242752541493
320;0.560283929568708
321;0.7406137530512305
322;0.2722555275418127
323;0.17081513186515518
324;0.12379040014044362
325;0.9925437730617201
326;0.9179762936630308
327;0.4587816585884308
328;0.5674766559408495
329;0.2410727687188251
330;0.521527949772088
331;0.118917948454911
332;0.8675105040551649
333;0.7289062937365407
334;0.9321515277590532
335;0.7066075198376208
336;0.626841411523038
337;0.4229738694424192
338;0.5842003838884097
339;0.647261479654959
340;0.9289288696688759
341;0.5362818239413248
342;0.7004771990056803
343;0.35167922117033235
344;0.6599733150683844
345;0.5176946635187637
346;0.56706366767864
347;0.5295651930889503
348;0.47699029688492944
349;0.6599960959026878
350;0.779826824958735
351;0.8986246049413006
352;0.2766120480724398
353;0.5961744566225088
354;0.41535711449471
355;0.3845252647921872
356;0.024742997627767105
357;0.7654551178086391
358;0.1363879423394858
359;0.9750041104726215
360;0.526063794340467
361;0.3355881381102468
362;0.5667198677275536
363;0.2699304103010839
364;0.48249301445085313
365;0.7292795462523095
366;0.4755701927162771
367;0.948530399137611
368;0.5705882345641431
369;0.03389901689460384
370;0.2846955303251548
371;0.5903571075463915
372;0.18759730512988737
373;0.4731553663514436
374;0.5589878244076226
375;0.9567326336757799
376;0.3431820487077013
377;0.6960263622045991
378;0.5065750102801503
379;0.3491196652263322
380;0.6950724957674771
381;0.6853072379775255
382;0.5552437167455284
383;0.8786925315333401
384;0.15904757503776912
385;0.8752078533357892
386;0.956060612549608
387;0.1589764507553656
388;0.9366874465333958
389;0.14453594909928835
390;0.66888892664582
391;0.07552194445248217
392;0.1490619018735838
393;0.6273918489798879
394;0.9051171448116431
395;0.1534822266318564
396;0.8872496624754551
397;0.1079746590269528
398;0.7538137841270107
399;0.6182446509316505
400;0.130962298224087
401;0.0318970935638081
402;0.8951757981008807
403;0.3438186262107229
404;0.05974882675062365
405;0.548218407842056
406;0.9896988984460504
407;0.022199760190016238
408;0.546259630867282
409;0.8641890010838845
410;0.2673534744009657
411;0.7243575861163687
412;0.6259749977719663
413;0.06350840662717705
414;0.593423654288721
415;0.5289357523389339
416;0.8775962016419309
417;0.6025432579922775
418;0.8540367150693828
419;0.5620686204146521
420;0.9933002021695158
421;0.4920802304948082
422;0.7682461808071875
423;0.0946336012081821
424;0.756982684385256
425;0.9260037950971421
426;0.09958794691844075
427;0.6145108763874917
428;0.34710278277647044
429;0.9900826132754563
430;0.9163128977715461
431;0.054631803673542256
432;0.4531081783536628
433;0.3050335827168954
434;0.11676037746224588
435;0.8771810287556686
436;0.36640743267461917
437;0.4762469200543499
438;0.22373318724970193
439;0.6964001485439532
440;0.3365528591838277
441;0.6910133935495071
442;0.5779662172408829
443;0.8481879271443511
444;0.5864635392228877
445;0.7448913175840461
446;0.9240764126003662
447;0.386696169232455
448;0.6467919029116486
449;0.7458395844655789
450;0.6076521920759168
451;0.9642817800359627
452;0.3720599112524733
453;0.649505034038631
454;0.4604985568873802
455;0.45652349824388183
456;0.2869310506863474
457;0.06631676506885165
458;0.02542912631684624
459;0.6638052575399219
460;0.3529366138335305
461;0.4684294964598643
462;0.9812130827915394
463;0.5275506854476261
464;0.6786009030289306
465;0.8516922730309306
466;0.06264984278526131
467;0.3152787222888318
468;0.557368627874004
469;0.48279989292626846
470;0.9341013275846727
471;0.002459545846705491
472;0.19780615169969396
473;0.8783145688768065
474;0.4269477086144958
475;0.028701598723478106
476;0.8850577426939149
477;0.16674707769718833
478;0.8747271240586209
479;0.35029976518864514
480;0.3015912993446338
481;0.11556984380359259
482;0.5672935642402629
483;0.41823684925933136
484;0.8389610279952519
485;0.34830972363528145
486;0.4518081616120476
487;0.29992920816854174
488;0.7750427220867744
489;0.26329351756697905
490;0.41207708368937823
491;0.31840529652049987
492;0.2472483271213387
493;0.16067017969099195
494;0.3617232426919651
495;0.3965059854518028
496;0.10228767069871803
497;0.690460377424534
498;0.8450361884603266
499;0.06050773477891347
500;0.2698803645605373
501;0.23051951342881982
502;0.8237059342794673
503;0.3654716421781322
504;0.6700841137913359
505;0.9997529553012626
506;0.9316431547613205
507;0.5973767004443995
508;0.1416015346512718
509;0.6428085198769999
510;0.2856286663670804
511;0.6877808062491569
512;0.9908829494941684
513;0.44733030946565155
514;0.3422272842811226
515;0.7848513533764071
516;0.3602451701865955
517;0.02944484215667942
518;0.6676212751456024
519;0.960644241237604
520;0.3474617529809577
521;0.8720585334138624
522;0.5043867489957397
523;0.46409609398288476
524;0.898111005516381
525;0.6865461491849422
526;0.6021344380993751
527;0.11152038977678791
528;0.03212414153811227
529;0.26730269707816356
530;0.2998346421746154
531;0.05960859632873072
532;0.11630866470597923
533;0.13259874191446375
534;0.4002459901875357
535;0.7386828929207728
536;0.6665093828653352
537;0.7747395215157903
538;0.4734259022567392
539;0.1897450928868678
540;0.800641869043661
541;0.5686912680071466
542;0.16700222147446409
543;0.639684667919672
544;0.753231722197939
545;0.12049240579872433
546;0.5593178368652086
547;0.7960260442335266
548;0.08689009791411884
549;0.07399340407114208
550;0.21219550062145498
551;0.9083901764361954
552;0.5171226080788464
553;0.7617579421128693
554;0.9747887207085306
555;0.8830177884217173
556;0.4146759851615861
557;0.128397590532784
558;0.9300933245986083
559;0.8896799810647662
560;0.1426219256038963
561;0.5710541453775289
562;0.9188700455974277
563;0.2950177616767238
564;0.45062764611444195
565;0.42722554050965944
566;0.5840161495318454
567;0.40788582860509237
568;0.19779582758139724
569;0.16044906850765595
570;0.5044748641144827
571;0.5937557720201581
572;0.7902741792183257
573;0.0803621727380498
574;0.10775468148892242
575;0.2516440124674981
576;0.328533777925722
577;0.5862807061274316
578;0.22605453583992263
579;0.5328042113906519
580;0.43510889208244197
581;0.16022452996853875
582;0.08966466424433239
583;0.6883599445170038
584;0.2548749181347577
585;0.5589668654071063
586;0.12347172310640453
587;0.8925833676640385
588;0.04585802980043219
589;0.6245786337797973
590;0.046687961783277165
591;0.9977444429532215
592;0.5516192596941207
593;0.7908810952184613
594;0.3548958580096059
595;0.18808908204993047
596;0.22038433646169264
597;0.142664235164078
598;0.2246487377145313
599;0.5717707802866521
600;0.8236656778065379
601;0.5119318874513681
602;0.7282510460675514
603;0.4954437923962418
604;0.8268911404534062
605;0.07522575310841495
606;0.08274738551756278
607;0.1564389829229813
608;0.07502367167619894
609;0.3030843161310083
610;0.5821963740641679
611;0.3426195831081338
612;0.5067277341010842
613;0.5504525047519074
614;0.3786033369978309
615;0.46962128479969534
616;0.682687228266767
617;0.49072831123631655
618;0.5155699743584589
619;0.6247998279871798
620;0.2815529929389289
621;0.7378569151605214
622;0.1397207097868346
623;0.5077043873780691
624;0.7682270829466987
625;0.47964485770163323
626;0.698621639810447
627;0.3642406440091134
628;0.2620560536263271
629;0.8847763952863557
630;0.8719245244427718
631;0.6479915246442188
632;0.036505356582640425
633;0.6514520303736802
634;0.23866895842904068
635;0.7173676660771148
636;0.101956104597139
637;0.460693946435487
638;0.13789205068287125
639;0.7492690646062463
640;0.26213404197470713
641;0.5735084964710387
642;0.974026320222745
643;0.14501969173472984
644;0.8737980610386175
645;0.6079254075237901
646;0.019924116675762438
647;0.3747674606883806
648;0.38128691159620864
649;0.05341056694894375
650;0.6685426596182117
651;0.331274927531784
652;0.3654229564118049
653;0.005081628557326301
654;0.5245353331072266
655;0.21116122609582277
656;0.21245700042806626
657;0.3891144321074528
658;0.6388300631069256
659;0.5228578336911147
660;0.31931299807536184
661;0.7036269507212011
662;0.06518287908887899
663;0.006328519122707044
664;0.9949523565618885
665;0.6628218303958757
666;0.14667830305585705
667;0.9440483901869922
668;0.49461135320717764
669;0.22813745890763748
670;0.9423160972880384
671;0.7850218254270999
672;0.07847288361847038
673;0.17714181267696738
674;0.7534335352556603
675;0.20725632603945643
676;0.4625340438181661
677;0.4927240341280922
678;0.6323730064301041
679;0.16021484871963365
680;0.6676386728143721
681;0.46677372316121557
682;0.00256648885655153
683;0.5741022849839944
684;0.782716273541694
685;0.3155997993767038
686;0.7991109960108576
687;0.07253405350539788
688;0.4438106175731401
689;0.7533470337286446
690;0.21567468156122305
691;0.944991347960591
692;0.5527242059200742
693;0.7516027804421979
694;0.9877645757687055
695;0.3673933997135481
696;0.12582725719735433
697;0.8759786705792756
698;0.6284353350321426
699;0.7237734678719465
700;0.7823877886684033
701;0.9131424680405861
702;0.6141734438427408
703;0.8293840867881902
704;0.18329054405800538
705;0.8234960686910402
706;0.13992527888476547
707;0.6769102025447961
708;0.6770543190007912
709;0.1284025307350145
710;0.5023573176541073
711;0.12068532191288972
712;0.4587557370336305
713;0.8694031668242342
714;0.7510393383421741
715;0.7817251606531923
716;0.13345198577362727
717;0.22066243380460548
718;0.8082454270751087
719;0.32613102988860265
720;0.5368736221935015
721;0.38796957979186686
722;0.1813632199485694
723;0.7755209363636545
724;0.5544917676686331
725;0.16103843631937798
726;0.5388185908517811
727;0.4553589667814777
728;0.61158144402648
729;0.27405485422780484
730;0.6557209043876443
731;0.900430760822417
732;0.5828589924977928
733;0.5065329479807247
734;0.31267659823300165
735;0.16866273757495587
736;0.06376058148097397
737;0.8702911036144971
738;0.9036481420609651
739;0.8267519060046692
740;0.04754908892776566
741;0.6270316270802055
742;0.9849645848226567
743;0.08753311865421032
744;0.1617051608901795
745;0.4116671140566541
746;0.4265534807292928
747;0.12708216628962532
748;0.06350707496418151
749;0.3651387959272041
750;0.5057917908500158
751;0.9318537494295327
752;0.2141482230309949
753;0.9296058581495698
754;0.7572691880893876
755;0.8322121630389795
756;0.13385927606226977
757;0.6798813630694963
758;0.03873109964742638
759;0.5835797978736242
760;0.5837396269666448
761;0.4168263878356787
762;0.19731950648829633
763;0.27414517830786966
764;0.1302274909598845
765;0.47777177515013025
766;0.897156906746577
767;0.022561384953298957
768;0.5838554219094025
769;0.6604216640786472
770;0.3282687696242377
771;0.9654055799737737
772;0.8975790416095465
773;0.22337880103572538
774;0.4644094648789625
775;0.7209714890602041
776;0.3662157531028605
777;0.5514658436555477
778;0.035623717081775697
779;0.2808809788250154
780;0.2668870613387406
781;0.08472609614633841
782;0.4308135435195827
783;0.11398726560502181
784;0.4536269658430283
785;0.3471365720988636
786;0.003144867849992572
787;0.36464458348170536
788;0.1225523584180872
789;0.3853643777510275
790;0.7618304699948037
791;0.9216232755774382
792;0.5740547415959647
793;0.32825923717855554
794;0.14089552984671183
795;0.8238874320299325
796;0.5252924193505716
797;0.2643559015815953
798;0.36352678320373677
799;0.043923841290745225
800;0.3127036780305378
801;0.14855732762573837
802;0.9102260409153955
803;0.94848795720202
804;0.7660068546864748
805;0.450198592528539
806;0.5064871111781948
807;0.14175288066497993
808;0.9839237053104714
809;0.8371939877943958
810;0.2194538675177815
811;0.026964836996351438
812;0.22388339316317574
813;0.5990524293813799
814;0.6740123611327773
815;0.4655253539838915
816;0.09169919568890839
817;0.30248764260767114
818;0.3698795993087862
819;0.9039415119642538
820;0.21897451589082784
821;0.6667375981572424
822;0.12485441521351293
823;0.39594258951546035
824;0.8691050048510721
825;0.15681198024205967
826;0.006089270386133272
827;0.5687869085402844
828;0.8918571264834463
829;0.9006668103735856
830;0.6161181282319836
831;0.2031051124350448
832;0.7827987432275001
833;0.32234107524705635
834;0.4234631362014396
835;0.7081666169368652
836;0.1967082791099105
837;0.697786205997863
838;0.36148811363527134
839;0.6280959329248041
840;0.7005311463887683
841;0.5257939510539725
842;0.6902293684061565
843;0.8782934809384683
844;0.8955279814315521
845;0.35708168403080665
846;0.7609945714619135
847;0.9875015686575168
848;0.9470393339627886
849;0.5174836900889181
850;0.0239825697024052
851;0.8787531954864306
852;0.9049107466878719
853;0.8774982642309316
854;0.5630159706091378
855;0.6949089987233522
856;0.8032308936909464
857;0.23000735233693093
858;0.08347572939498693
859;0.34206241718306596
860;0.43286217029011886
861;0.9459479357892713
862;0.15571428795198672
863;0.7606719413370542
864;0.8732877107361015
865;0.8540316820610468
866;0.39193474057105937
867;0.2627251986729703
868;0.7184484311893105
869;0.08437701863711766
870;0.9768412003701361
871;0.9994931875549745
872;0.7555315115184719
873;0.2729008724642469
874;0.68843498588785
875;0.833401093066513
876;0.8864149012849527
877;0.7310870315291682
878;0.12258684446932433
879;0.09648600941589902
880;0.40908827270971904
881;0.7307114079702229
882;0.9233614547677758
883;0.11687564894897806
884;0.8935601743491578
885;0.08003581738401211
886;0.7825619987507674
887;0.7922979698647097
888;0.15144455127902223
889;0.1856053272248025
890;0.391438131581764
891;0.24964777730647392
892;0.9798365827978819
893;0.6628385496890475
894;0.1985349487870558
895;0.7022718838261685
896;0.623036535005391
897;0.578507905341236
898;0.37981494391546333
899;0.5027754851102614
900;0.2614052871624831
901;0.938925600447462
902;0.29648106881134784
903;0.7107664136484366
904;0.6766569414269537
905;0.3056056095509473
906;0.8463059539564312
907;0.06366108264335457
908;0.6788154262756905
909;0.4232273391027187
910;0.2483516317144734
911;0.3400007278997643
912;0.9611416572315553
913;0.004065856931176559
914;0.5871055270174178
915;0.5135659150529975
916;0.17394482147391377
917;0.1883873986021104
918;0.01734452066423353
919;0.2519492137405904
920;0.30840454310016885
921;0.7439008760289331
922;0.6390987114182279
923;0.08253715228631076
924;0.08890635563063887
925;0.5745360268703207
926;0.5403267213742577
927;0.393076357952882
928;0.9169664917800239
929;0.7382335842423199
930;0.9225664711766852
931;0.48595305531730204
932;0.6395626669751501
933;0.9481794984997332
934;0.29056478550534237
935;0.8860754453528294
936;0.7731829121505578
937;0.8590000260829755
938;0.23408832767000487
939;0.6538381400812041
940;0.535873514742711
941;0.49500136174173937
942;0.27715860000120585
943;0.8035652091135874
944;0.4490382943128517
945;0.9573558423175347
946;0.7816888788330443
947;0.08500857334816758
948;0.03755127426140148
949;0.06129639109735452
950;0.7768430145476535
951;0.4930080900791647
952;0.5756697542135483
953;0.5866502650232608
954;0.48352688109266395
955;0.7454772499248578
956;0.36333562168581457
957;0.7121250044073716
958;0.5590919300662571
959;0.3686360447428003
960;0.26600070975568846
961;0.9400083636288769
962;0.9065853908769868
963;0.35043519988634453
964;0.8667225071368346
965;0.7623437195331065
966;0.3666504513097397
967;0.43436718504361604
968;0.9983905591721135
969;0.5180230839748512
970;0.7041824658361329
971;0.28849599161274
972;0.14894786984734432
973;0.688485092024457
974;0.4377782574073389
975;0.8664874770720213
976;0.493085127203185
977;0.07769885204032134
978;0.11800135622022567
979;0.13315357975539177
980;0.09707539516524777
981;0.9930030430749599
982;0.14343295097397235
983;0.5481140495123301
984;0.997736211282004
985;0.6110387769602259
986;0.4777952405485105
987;0.49579435161704777
988;0.520425411815102
989;0.07557624001352603
990;0.674399237462791
991;0.3063331985383748
992;0.5356727212218421
993;0.7314216248577331
994;0.5185711519968934
995;0.58555355719543
996;0.501290072929127
997;0.591506336440278
998;0.383783358090371
999;0.17804272980857772
