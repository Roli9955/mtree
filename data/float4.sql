/* contrib/mtree_gist/data/float4.sql */

DROP TABLE IF EXISTS FLOAT4_TEST CASCADE;
CREATE TABLE FLOAT4_TEST (
  id  INTEGER,
  val mtree_float
);
COPY FLOAT4_TEST(id, val) FROM '/home/postgres/test_files/float4.csv' DELIMITER ';' CSV HEADER;

DROP INDEX IF EXISTS FLOAT4_TEST_IDX CASCADE;
CREATE INDEX FLOAT4_TEST_IDX ON FLOAT4_TEST USING GiST (val gist_mtree_float_ops);

SET enable_seqscan TO OFF;

SELECT COUNT(*) FROM FLOAT4_TEST;

SELECT * FROM FLOAT4_TEST;

SELECT id, val, (val <-> '0') AS dst FROM FLOAT4_TEST ORDER BY (val <-> '0');

SELECT * FROM FLOAT4_TEST WHERE val #<# '0';
