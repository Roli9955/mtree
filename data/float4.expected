DROP TABLE
CREATE TABLE
COPY 4
DROP INDEX
CREATE INDEX
SET
 id |    val    
----+-----------
  0 | -0.999999
  1 | 0.000000
  2 | 0.999999
  3 | 1.000000
(4 rows)

 id |    val    |   dst    
----+-----------+----------
  1 | 0.000000  |        0
  0 | -0.999999 | 0.999999
  2 | 0.999999  | 0.999999
  3 | 1.000000  |        1
(4 rows)

 id |   val    
----+----------
  1 | 0.000000
(1 row)

